src/components/Field_Input.tsx

```diff
@@ -0,0 +1,66 @@
+import Image from "next/image";
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  placeholder?: string;
+  icon?: string;
+  required?: boolean;
+}
+
+/*
+This is a component for the HTML `<input>' tag
+Input:
+  label: The label displayed on the text box
+  placeholder: The placeholder message displayed on the text box
+  icon: An optional Image path to display an .svg file
+Output:
+  A input text box that is compatible w/ React-hook-forms
+*/
+export default function Field_Input<T extends FieldValues = FieldValues>({
+  label,
+  placeholder = "",
+  icon = "",
```

Comment:
You don't have to specify a default value for these. I think it would be cleaner to check if it the prop was provided if you removed it and checked with
```suggestion
!placeholder && `Enter ${props.name}`
```
or
```suggestion
!placeholder ? `Enter ${props.name}` : ""
```